designs/octoplushy/src/head.mjs

```diff
@@ -613,15 +613,16 @@ const options = {
     min: 5,
     max: 500,
     menu: 'style',
+    // eslint-disable-next-line no-unused-vars
```

Comment:
You can use `toAbs: (val, _, mergedOptions) => ` if you don't need the `options` variable and then don't need to disable the lint warning.

Reply:
Done

Reply:
Use `_`, not `{}`, or you get another warning. 🙄

Reply:
I added an ignore line ;-)